web/src/app/services/app-state.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AppStateService {
+// tslint:disable-next-line: variable-name
+  private _authState: boolean;
+
+  constructor() {
+    if (localStorage.getItem('authState') != null) {
+      this._authState =  (localStorage.getItem('authState') as any) as boolean;
```

Comment:
Not sure the `as any` is required, as it will be `any` by default

Reply:
The default return type of localstorage is String. So typescript throws an error we do explict conversion to boolean directly. The suggestion provided  by typescript is to convert it to any and to convert it to boolean.

Reply:
Thinking about it, Angular Firebase should take care of the local storage for us? Can you share a link to the docs please. Thanks

Reply:
Example https://angularfirebase.com/lessons/google-user-auth-with-firestore-custom-data/

Reply:
Hi Eddie,
I am currently working on this issue. The above link was useful. Thanks. I will fix the issue and raise PR. Then will move to designing the dashboard. Will keep you posted about the progress.